src/Mock/MockedDBALConnection.php

```diff
@@ -52,8 +52,10 @@ public function query(string $sql): Result
 
     /**
      * {@inheritdoc}
+     *
+     * @return mixed
      */
-    public function quote($input, $type = ParameterType::STRING)
+    public function quote($input, $type = ParameterType::STRING)/*: mixed // <--- from php 8*/
```

Comment:
We can perfectly increase the PHP dependency version.
Even add 8.1 in the circleci raw.

Reply:
yes for Drift, but no for us, since currently for MSP CHallenge we are at php 7.4. So if reactphp-dbal would require 8.1 this would be a breaking change for projects already using it, so a major change, so we should then release that a new major version? 
So I suggest to keep it compatible with the current  php , and do a seperate php 8 release later